### ISSUE TITLE
[NSURL URLWithString:] not converted correctly.

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#64]
The same is likely for most of the factory methods, which are usually translated like this:

- If the class has the same suffix as prefix of the class method, it's considered an initializer.

**Comment by drkameleon** [comment]
That's a good catch I guess... Thanks for reminding me! I'll include it in the fix of the upcoming update. ;-)

**Comment by drkameleon** [comment]
Fixed (I hope lol) as of the upcoming 1.9.

**Example:**

```objective-c
NSURL* u = [NSURL URLWithString:@"http://www.iswift.org"];
```

**Output:**

```swift
var u: NSURL = NSURL(string:"http://www.iswift.org")
```

**Comment by charlieMonroe** [comment]
Awesome, thanks!
